src/d3a/models/myco_matcher/myco_internal_matcher.py

```diff
@@ -58,18 +58,21 @@ def get_matching_algorithm():
     def match_recommendations(self, **kwargs):
         """Request trade recommendations and match them in the relevant market."""
         for area_uuid, area_data in self.area_uuid_markets_mapping.items():
```

Comment:
```suggestion
        for area_data in self.area_uuid_markets_mapping.values():
```

Reply:
Will fix this, however keep in mind that this might change again during this task https://gridsingularity.atlassian.net/browse/D3ASIM-3593